config.js

```diff
@@ -80,12 +80,14 @@ config.cookie = {
 /*!
  * The servers related configuration
  *
+ * @property  {String}      serverHostname          The hostname on which the web servers should be listening. See https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback
  * @property  {String}      adminHostname           The hostname on which the admin UI will be made available
  * @property  {Number}      adminPort               The port on which the admin API endpoints will be made available
  * @property  {Number}      appsPort                The port on which the regular application API endpoints will be made available
  * @property  {String}      shibbolethSPHost        The hostname on which the Shibboleth Service Provider software will be made available
  */
 config.servers = {
+    'webServerHostname': '0.0.0.0',
```

Comment:
I used `hostname` as that is [what node.js uses in their own documentation](https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback) :(

I looked around a bit and it looks like:
 * [Apache uses `address`](http://httpd.apache.org/docs/2.2/bind.html#page-header)
 * [Nginx uses `address`](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen)
 * [Elasticsearch uses `host` in their config, `address` in their documentation](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration.html#settings)
 * [RabbitMQ uses `address` in their config, `interface` in their documentation](https://www.rabbitmq.com/configure.html)

What if we called it `expressListenAddress`? That way it contains:
 - `express, so you know it's about the ExpressJS http server
 - `ListenAddress`, so even if you don't know what express is, you know it's something that will be listening on a specific address.